### ISSUE TITLE
Add Markdown syntax highlighting

### ISSUE THREAD

**Issue by krstp** [zed-industries/community#1315]
### Check for existing issues

- [X] Completed

### Describe the feature

Markdown `.md` files syntax highlighting is missing.

**Comment by jansol** [comment]
I do get syntax highlighting in `README.md` files? And Markdown is in the list of known languages so you can specify it manually for files where it is not automatically detected. (press cmd+k, release both keys and press m to toggle the language selector... or use the button in the statusbar)

**Comment by krstp** [comment]
Indeed, I do get syntax highlighting; sorry for that, I was viewing wrong file that did not have too many "decorations". Needless to say, would be great for any type of syntax to be able to change coloring in particular different block coloring etc. I know, I know, is coming, it is on the roadmap, just want to reiterate that would be desirable feature. Thanks for all the awesome work you do, for now ZED became my no.1 editor (being diehard sublime user). Loving it and waiting for more features!
